api/storage/version_endpoint_storage.go

```diff
@@ -54,7 +54,7 @@ func (v *versionEndpointStorage) Get(uuid uuid.UUID) (*models.VersionEndpoint, e
 
 func (v *versionEndpointStorage) Save(endpoint *models.VersionEndpoint) error {
 	sanitizeEndpoint(endpoint)
-	return v.db.Save(&endpoint).Error
+	return v.db.Session(&gorm.Session{FullSaveAssociations: true}).Save(&endpoint).Error
```

Comment:
I have tried using GROM Clauses but to no avail, since clauses cannot be used to update the associations. @deadlycoconuts @krithika369

Reply:
@ariefrahmansyah will this also trigger a save on the `environments` table?

Instead of using `FullSaveAssociations`, I'm thinking if we should explicitly issue `Save` on the transformer record. Like:

```golang
func (v *versionEndpointStorage) Save(endpoint *models.VersionEndpoint) error {
    sanitizeEndpoint(endpoint)
    if err := v.db.Save(&endpoint).Error; err != nil {
        return err
    }
    
    return v.db.Save(endpoint.Transformer).Error
}
```